torch/_dynamo/source.py

```diff
@@ -9,22 +9,13 @@
 from .bytecode_transformation import create_call_function, create_instruction
 from .utils import enum_repr
 
-# It shouldn't be supported to construct an NNModuleVariable inside an FSDP module,
-# so those cases are omitted intentionally
 _GUARD_SOURCE_NN_MODULE = {
     GuardSource.LOCAL: GuardSource.LOCAL_NN_MODULE,
     GuardSource.GLOBAL: GuardSource.GLOBAL_NN_MODULE,
     GuardSource.LOCAL_NN_MODULE: GuardSource.LOCAL_NN_MODULE,
     GuardSource.GLOBAL_NN_MODULE: GuardSource.GLOBAL_NN_MODULE,
-}
-
-_GUARD_SOURCE_FSDP_MODULE = {
```

Comment:
i like that this gets collapsed. but i also don't understand/remember why i made it separated before.

Reply:
I don't remember either...